docs/asyncapi.yaml

```diff
@@ -324,6 +324,8 @@ components:
                 $ref: '#/components/schemas/SchemeCode'
               value:
                 $ref: '#/components/schemas/ValueDecimal'
+              deliveryBody:
+                $ref: '#/components/schemas/DeliveryBody'
```

Comment:
Would we also need to add convergence to the asyncapi doc?

Reply:
Or because it's not required it's not necessarily needed?

Reply:
It should but it's not technically in scope of this change.  Given it's such a simple change I'd normally just do it anyway, but looking through the async api it's outdated in other areas too.  It also seems to have some errors in it.  I think instead what we should do is create a continuous improve ticket and deal with tidying the rest of this spec as it's own item of work as it could take some time.

Reply:
Ticket created -> https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/175017

Reply:
Could the name be change to AsyncAPI to make it more clear?

Reply:
Which name?

Reply:
Sorry, I meant the name of the ticket

Reply:
Gotcha, I was going to add it as part of the acceptance criteria for the ticket.  I can see a lot of things wrong so just going to list them all in the detail

Reply:
Sounds good